src/SportsData.Api/Infrastructure/Data/Canonical/Sql/GetMatchupsForSeasonWeek.sql

```
@@ -63,13 +63,61 @@ inner join public."GroupSeason" gsAway on gsAway."Id" = fsAway."GroupSeasonId"
+  -- SeasonPollWeek.SeasonWeekId — those links are unreliable
+  -- (off-by-one late season, NULL for preseason/final).
+  SELECT spwe."Current"
+  FROM public."SeasonPollWeekEntry" spwe
+  WHERE spwe."SeasonPollWeekId" = (
+      SELECT spw."Id"
+      FROM public."SeasonPollWeek" spw
+      INNER JOIN public."SeasonPoll" sp ON sp."Id" = spw."SeasonPollId"
+      WHERE sp."SeasonYear" = fsAway."SeasonYear"
+        AND spw."Type" IN ('ap', 'cfp')
+        AND spw."DateUtc" < (SELECT wk."StartDate" + INTERVAL '5 days'
+                             FROM public."SeasonWeek" wk WHERE wk."Id" = sw."Id")
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)